dojo/settings/settings.dist.py

```diff
@@ -1474,6 +1474,7 @@ def saml2_attrib_map_format(dict):
     'Nosey Parker Scan': DEDUPE_ALGO_UNIQUE_ID_FROM_TOOL_OR_HASH_CODE,
     'Bearer CLI': DEDUPE_ALGO_HASH_CODE,
     'Wiz Scan': DEDUPE_ALGO_HASH_CODE,
+    'Rapplex Scan': DEDUPE_ALGO_UNIQUE_ID_FROM_TOOL,
```

Comment:
Based on my other comment related to reduplication, I believe that setting a hash code would be preferable here. Looking at the report, an how the dojo finding is structured, I think the following fields could be the most effective:
```
'Rapplex Scan': ["title", "severity", "endpoints"], 
```
And then update the algorithm used
```suggestion
    'Rapplex Scan': DEDUPE_ALGO_HASH_CODE,
```
```

Reply:
Deduplication algorithm has changed. unique_id_from_tool -> hash_code